### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### Changes made

- Added styles to my app according to the design of the given [zeplin project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -20,12 +20,14 @@ const AddBook = () => {
...
-      <h2>ADD NEW BOOK</h2>
-      <input className="title" type="text" placeholder="Book title" value={title} onChange={(e) => setTitle(e.target.value)} />
-      <input className="author" type="text" placeholder="Book author" value={author} onChange={(e) => setAuthor(e.target.value)} />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="adding">
+      <h2 className="new-title">ADD NEW BOOK</h2>
+      <form className="newbook" onSubmit={actionSubmit}>
+        <input className="title" type="text" placeholder="Book title" value={title} onChange={(e) => setTitle(e.target.value)} />
+        <input className="author" type="text" placeholder="Book author" value={author} onChange={(e) => setAuthor(e.target.value)} />
```
Comment (ahmetbozaci) [inline src/components/AddBook.js:27]: - [ ] Input values are not showing when ı write something in it. Please fix it.
- [ ] Kindly give space end of the form.

### ORIGINAL DESIGN
image

File: src/pages/Navbar.js [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
     <div className="nav-links">
-      <a className="link" href="/">Books</a>
-      <a className="link" href="/Category">Category</a>
+      <a className="books" href="/">BOOKS</a>
+      <a className="categories" href="/Category">CATEGORIES</a>
     </div>
-  </div>
+    <div className="oval">
+      <Icon icon="bi:person-fill" className="mask" />
+    </div>
```
Comment (ahmetbozaci) [inline src/pages/Navbar.js]: - [ ] Please make center the icon inside the circle.
image

File: src/index.scss [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+div .line {
+  width: 70rem;
+  height: 0.125rem;
+  border: 1px solid $greyBorder;
+  margin: 2.5rem 4rem 1.813rem 5.9rem;
+}
+
+.adding {
+  display: flex;
```
Comment (ahmetbozaci) [inline src/index.scss]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [ ] This is just an example if you use duplicate styles please combine them.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Book.js [hunk 4]
```diff
@@ -11,10 +11,20 @@ const Book = (props) => {
...
+        <p className="bookAuthor">{book.author}</p>
+        <button type="button" className="onBook comment">Comment</button>
+        <button type="button" onClick={() => deleteBook(book.id)} className="remove">Remove</button>
+        <button type="button" className="onBook">Edit</button>
+      </div>
+      <div className="chapterUpdate">
+        <p className="chapterHeading">CURRENT CHAPTER</p>
+        <p className="chapter">Chapter 1</p>
+        <button type="button" className="progress">UPDATE PROGRESS</button>
+      </div>
```
Comment (ahmetbozaci) [inline src/components/Book.js]: - [ ] To follow the design please add a progress circle.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @kemigabocatherine  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by arslanbisharat** [comment]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**